src/sm/Materials/LatticeMaterials/latticestructuralmaterial.C

```diff
@@ -61,17 +62,44 @@ LatticeStructuralMaterial :: hasMaterialModeCapability(MaterialMode mode) const
     return mode == _3dLattice || mode == _2dLattice || mode == _1dLattice;
 }
 
+int
+LatticeStructuralMaterial :: giveIPValue(FloatArray &answer,
+                                         GaussPoint *gp,
+                                         InternalStateType type,
+                                         TimeStep *atTime)
+{
+    LatticeMaterialStatus *status = static_cast< LatticeMaterialStatus * >( this->giveStatus(gp) );
+
+    if ( type == IST_LatticeStress ) {
+        answer.resize(6);
+        answer.zero();
+        status->giveLatticeStress();
+    } else if  ( type == IST_LatticeStrain ) {
+        answer.resize(6);
+        answer.zero();
+        status->giveLatticeStrain();
```

Comment:
and this
```
answer = status->giveLatticeStrain();
```

Reply:
Thanks. Must have been too late when I typed this.